ansible/roles/grafana/defaults/main.yml

```diff
@@ -0,0 +1,182 @@
+---
+# .. vim: foldmarker=[[[,]]]:foldmethod=marker
+
+# debops.grafana default variables
+# ================================
+
+# .. contents:: Sections
+#    :local:
+#
+# .. include:: ../../../../includes/global.rst
+
+
+# Domain name configuration [[[
+# -----------------------------
+
+# .. envvar:: grafana__hostname [[[
+#
+# The hostname part of the Grafana installation
+grafana__hostname: 'grafana'
```

Comment:
Why not use `{{ ansible_hostname }}` instead? Actually, this variable is not used anywhere besides `grafana__fqdn`, I think that it can be safely removed.

Reply:
done. removed the variable